web/public/templates/dojos/dojo-detail.dust

```diff
@@ -8,6 +8,7 @@
             <b ng-bind-html="dojo.name"></b>
             <span ng-if="dojo.verified === 1" class="verification-icon"><span class="cd-success-text glyphicon glyphicon-ok-sign"></span> {@i18n key=" Verified!"/}</span>
             <span ng-if="dojo.verified != 1" class="verification-icon"><span class="cd-warning-text glyphicon glyphicon-remove-sign"></span> {@i18n key=" Not verified"/}</span>
+            <span ng-if="dojo.taoVerified === 1" class="verification-icon"><span class="cd-success-text glyphicon glyphicon-ok-sign"></span> {@i18n key=" Tao verified"/}</span>
```

Comment:
The icon is the same indeed, but wording is different. As agreed with @tangentfairy, this is more of a placeholder for now, but I'll try to find a better icon